pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aead_composite_encrypt.go

```diff
@@ -94,6 +98,7 @@ func (e *ECDHESAEADCompositeEncrypt) Encrypt(plaintext, aad []byte) ([]byte, err
 	tagOffset := len(ctAndTag) - tagSize
 
 	encData := &EncryptedData{
+		EncAlg:     A256GCM, // TODO support 25519 content encryption as well
```

Comment:
Create a story and link it here for this TODO?

Reply:
added